src/include/execution/compiler/operator/limit_translator.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include <utility>
+#include <vector>
+#include "execution/compiler/operator/operator_translator.h"
+#include "execution/compiler/translator_factory.h"
+#include "planner/plannodes/limit_plan_node.h"
+
+namespace terrier::execution::compiler {
+
+/**
+ * Limit Translator
+ * TODO(Amadou): The limit should be pushed down to the child operators. That's only being done with OrderBy.
```

Comment:
What's the purpose of the TODO? Limits/offsets cannot always be pushed down, so we'll always need this implementation.

Reply:
Removed.